skfp/utils/parallel.py

```diff
@@ -101,10 +103,22 @@ def run_in_parallel(
     data_batch_gen = (data[i : i + batch_size] for i in range(0, len(data), batch_size))
     num_batches = len(data) // batch_size
 
-    if verbose > 0:
-        parallel = ProgressParallel(n_jobs=n_jobs, total=num_batches)
+    if isinstance(verbose, int):
+        tqdm_settings = {
+            "total": num_batches,
+            "desc": "Processing",
+            "disable": verbose == 0,
+        }
     else:
+        assert isinstance(verbose, dict)
+        tqdm_settings = verbose.copy()
+        tqdm_settings["total"] = num_batches
+        tqdm_settings["disable"] = verbose.get("disable", False)
```

Comment:
Note that `assert` can get erased with Python optimizations. It should not be used for production code, just for tests. Use `elif isinstance(verbose, dict)`, and raise ValueError in `else`

Reply:
Done